lib/puppet_x/gitlab/dumper.rb

```diff
@@ -0,0 +1,122 @@
+# The Dumper class was blindly copied from https://github.com/emancu/toml-rb/blob/v2.0.1/lib/toml-rb/dumper.rb
```

Comment:
I think we need to make it clear that this part of the code is [MIT license](https://law.stackexchange.com/questions/6081/can-i-bundle-mit-licensed-components-in-a-apache-2-0-licensed-project#:~:text=Both%20the%20Apache%20License%20and,as%20required%20by%20the%20license.)?

Reply:
Right oO What's the correct way to do it? Adding the [MIT License](https://github.com/emancu/toml-rb/blob/v2.0.1/LICENSE) as a header of that file?

Reply:
yeah, I think that'll be fine.  And maybe a note in the README.

Reply:
maybe also append it to https://github.com/voxpupuli/puppet-gitlab_ci_runner/blob/master/LICENSE ?

Reply:
I've added it as an header in `lib/puppet_x/gitlab/dumper.rb`  and a note in README.md. Putting two licenses in the same LICENSE file feels wrong to me :)